alica_tests/etc/plans/SameBehaviourInParallelTestPlan.pml

```diff
@@ -0,0 +1,310 @@
+{
+  "blackboard": [
+    {
+      "access": "protected",
+      "comment": "",
+      "id": 549074179013053940,
+      "key": "resultOne",
+      "type": "int64"
+    },
+    {
+      "access": "protected",
+      "comment": "",
+      "id": 3537510304698959535,
+      "key": "resultTwo",
+      "type": "int64"
+    },
+    {
+      "access": "input",
+      "comment": "",
+      "id": 3628916307787761189,
+      "key": "baseValue",
+      "type": "int64"
+    },
+    {
+      "access": "protected",
+      "comment": "",
+      "id": 4053503531230885495,
+      "key": "EntryState2FirstCheckState",
+      "type": "bool"
+    }
+  ],
+  "comment": "",
+  "entryPoints": [
+    {
+      "comment": "",
+      "id": 682826694350781678,
+      "isDynamic": false,
+      "maxCardinality": 2147483647,
+      "minCardinality": 0,
+      "name": "",
+      "plan": 1467955996026918944,
+      "positionWeb": {
+        "x": 200,
+        "y": 212
+      },
+      "state": 2478721930275583422,
+      "successRequired": false,
+      "task": "taskrepository.tsk#1225112227903"
+    }
+  ],
+  "frequency": 0,
+  "id": 1467955996026918944,
+  "implementationName": "",
+  "inheritBlackboard": false,
+  "libraryName": "alica-tests",
+  "masterPlan": false,
+  "name": "SameBehaviourInParallelTestPlan",
+  "preCondition": null,
+  "runtimeCondition": null,
+  "states": [
+    {
+      "comment": "",
+      "confAbstractPlanWrappers": [],
+      "entryPoint": null,
+      "id": 31788376439727489,
+      "inTransitions": [
+        1251767179803755726
+      ],
+      "name": "SecondCheckState",
+      "outTransitions": [
+        4124693711202717699
+      ],
+      "parentPlan": 1467955996026918944,
+      "positionWeb": {
+        "x": 944,
+        "y": 200
+      },
+      "type": "State",
+      "variableBindings": []
+    },
+    {
+      "comment": "",
+      "confAbstractPlanWrappers": [],
+      "entryPoint": null,
+      "id": 821449835430427634,
+      "inTransitions": [
+        4124693711202717699
+      ],
+      "name": "SuccessState",
+      "outTransitions": [],
+      "parentPlan": 1467955996026918944,
+      "positionWeb": {
+        "x": 1202,
+        "y": 200
+      },
+      "postCondition": null,
+      "success": true,
+      "type": "TerminalState",
+      "variableBindings": []
+    },
+    {
+      "comment": "",
+      "confAbstractPlanWrappers": [
+        {
+          "abstractPlan": "AddToValueBeh.beh#1629144600464136880",
+          "comment": "",
+          "configuration": null,
+          "id": 3188735900091882738,
+          "keyMapping": {
+            "input": [
+              {
+                "childKey": "valueAddition",
+                "parentKey": null,
+                "value": 2
+              },
+              {
+                "childKey": "valueBase",
```

Comment:
again shown as red in both plan & behaviour. Is it designer problem?

Reply:
Designer has issues with default values of 0, using a value != 0 fixes the issue. I will make a wrike task for it